packages/frontend/app/components/password-validator.hbs

```diff
@@ -0,0 +1,59 @@
+{{#if (has-block)}}
+  {{#let (unique-id) as |templateId|}}
+    <div class="password-validator" data-test-password-validator>
+      <label for="password-{{templateId}}">
+        {{t "general.password"}}:
+      </label>
+      {{yield
+        this.password
+        this.setPassword
+        this.addErrorDisplayFor
+        this.keyboard
+        this.hasErrorForPassword
+        this.passwordStrengthScore
+      }}
+    </div>
+  {{/let}}
+{{else}}
+  {{#let (unique-id) as |templateId|}}
+    <div class="password-validator" data-test-password-validator>
+      <label for="password-{{templateId}}">
+        {{t "general.password"}}:
+      </label>
+      <input
+        id="password-{{templateId}}"
+        type="text"
+        value={{this.password}}
+        {{on "input" (pick "target.value" this.setPassword)}}
+        {{on "keyup" (queue (fn this.addErrorDisplayFor "password"))}}
+        {{on "keyup" this.keyboard}}
+        data-test-password-input
+      />
+      {{#if (this.hasErrorForPassword)}}
```

Comment:
```suggestion
     {{#if this.hasErrorForPassword}}
```

Reply:
This worked to fix the actual app, but tests still don't show the error message :-/
![Screenshot 2024-11-12 at 11 09 57 AM](https://github.com/user-attachments/assets/3babfd8b-6daa-45ab-b313-dbb61eb60c30)

Reply:
@stopfstedt OK, so the error message appears in the Integration test for `PasswordValidator`, but when it's used on the `NewUser` component, it does not. Seems it's being swallowed when trying to move upwards, but not sure how to fix.